genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/MonitorJobAction.java

```diff
@@ -72,12 +73,30 @@ protected Events executeStateAction(final ExecutionContext executionContext) {
 
         log.info("Job process completed with final status {}", finalJobStatus);
 
+        // TODO: Likely want to clean this up as it only contains a few cases.
```

Comment:
Alternative, have constant format strings in addition to constant strings.
i.e. this one could be `"Job process completed with final status: %s"`

Reply:
It works but won't match what is in the current V3 responses (if we want to maintain backwards capability) also that string doesn't capture things like "Killed due to timeout" or "Killed due to std out too large"